### ISSUE TITLE
Ensure additional questions can be answered by the documentation

### ISSUE THREAD

**Issue by EOstridge** [department-for-transport-BODS/bods-data-extractor#64]
**Is your feature request related to a problem? Please describe.**

As a data consumer, I would like further information to be provided by the documentation to make bus data more accessible. 

**Describe the solution you'd like**

It would be useful if the documentation could answer the following questions:
- What is the difference between a service and a line? 
- What is the OTC database and how does it relate? 
- What are PTI standards? What is the link for more info? 
- Does BODS include data for all operators? 
- What makes a service code invalid? What are the consequences for including invalid service codes in analysis? 
- What does it mean if a service has no licence number? What are the consequences? 
- How are operator (and presumably NOC), service, line, timetable, stop (and presumably ATCO) and anything else related? - Users to be able to visualise the structure of the data. 
- How is the service code derived? What would this be called in layman terms? 
- How do the different levels of data download relate to each other?  What is an example question I could answer from each type of download - and how might I go about answering it? 
- What does it mean if a timetable for a service code has a duplicate sequence number? 
- Is there an easy way for me to obtain all the services which serve one particular stop? 
- What are the VJ columns? How do they relate to each other, if at all? How do I know which sequence number is for which VJ? 
- How do I know what days a timetable runs? 
- Is it possible to see what non-standard stop level files could not be processed (part of the roadmap caveats)? 
- Why might the BODS/OTC comparison be useful to a user? 
- For services by area OTC/BODS, what if service operates in more than one area, or does this not happen? 
- How is data quality score calculated? If this is explained elsewhere in BODS documentation then we need to link to it.

**Comment by adamakram1** [comment]
Create an additional section in documentation "FAQ"

**Comment by adamakram1** [comment]
Try link questions to relevant sections in documentation

**Comment by adamakram1** [comment]
Will review in Dev Meeting
